ohttp.go

```diff
@@ -418,6 +418,30 @@ func (g Gateway) Config(keyID uint8) (PublicConfig, error) {
 	return PublicConfig{}, fmt.Errorf("unknown keyID %d", keyID)
 }
 
+func (g *Gateway) DropConfig(keyID uint8) error {
+	if _, ok := g.keyMap[keyID]; ok {
+		delete(g.keyMap, keyID)
+		newKeys := make([]uint8, 0)
+		for keyId := range g.keyMap {
+			newKeys = append(newKeys, keyId)
+		}
+		g.keys = newKeys
+
+	}
+	return fmt.Errorf("unknown keyID %d", keyID)
```

Comment:
you likely want to return in the above `if` block.

rn, you're returning an error no matter what.